clients/csharp-uwp/docs/sample-app-walkthrough.md

```diff
@@ -0,0 +1,105 @@
+# Understanding the sample app
+This is a detailed breakdown of the steps to build a Windows voice assistant along with an explanation of how the UWP Voice Assistant sample implements each step. Please read the [readme](https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/clients/csharp-uwp/README.md) for this sample for more information on MVA, DLS, and the UWP Voice Assistant Sample.
```

Comment:
MVA has always been an internal acronym. Are we going to use it externally? is it needed? can't we simply refer to the Windows API namespace, which is the public term developers see?

Reply:
DLS - When first shown in this document, acronyms should be expanded.

Reply:
Travis created a public vanity URL for https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/clients/csharp-uwp. Consider using that instead of all the links below. It is likely that we will do more code move in this GitHub repo, so it will be easy to update just the vanity URL and not all the links here.

Reply:
> Travis created a public vanity URL for https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/clients/csharp-uwp. Consider using that instead of all the links below. It is likely that we will do more code move in this GitHub repo, so it will be easy to update just the vanity URL and not all the links here.

Is there a way to append a path to it? http://aka.ms/mva/sample/UWPVoiceAssistantSample/DialogManager.cs for example doesn't work.